tests/src/JIT/Performance/CodeQuality/BenchmarksGame/revcomp/revcomp.csharp-1.cs

```diff
@@ -53,40 +53,18 @@ struct Index {
 
    static string FindInput(string s)
    {
-       string CoreRoot = System.Environment.GetEnvironmentVariable("CORE_ROOT");
-
-       if (CoreRoot == null)
-       {
-           Console.WriteLine("This benchmark requries CORE_ROOT to be set");
-           return null;
-       }
 
        string inputFile = s ?? InputFile;
 
-       // Normal testing -- input file will end up next to the assembly
-       // and CoreRoot points at the test overlay dir
-       string[] pathPartsNormal = new string[] {
-           CoreRoot, "..", "..", "JIT", "Performance",
-           "CodeQuality", "BenchmarksGame", "revcomp", "revcomp", inputFile
-       };
-
-       string inputPathNormal = Path.Combine(pathPartsNormal);
-
-       // Perf testing -- input file will end up next to the assembly
-       // and CoreRoot points at this directory
-       string[] pathPartsPerf = new string[] { CoreRoot, inputFile };
-
-       string inputPathPerf = Path.Combine(pathPartsPerf);
+       // Input file will end up next to the assembly
+       string currentDirectory = AppContext.BaseDirectory;
+       string[] inputPathParts = new string[] {currentDirectory, inputFile};
```

Comment:
Yeah you're right, I actually had it that way initially, but then changed to this to see if the error was caused by some weird behavior in C# array initializers - I want to see if the Pri-1 test job passes in CI, then I'll make that change after